NuCivic/dkan_starter
# Remove deprecated modules from features master CIVIC-5614

Issue: CIVIC-5614

## Description
We've removed several modules from dkan core in 1.13 but don't have a hook_update to remove those modules from the system table of client sites and two of them are still in features master.

## QA Tests
Run `drush cc all`  you should not see: `The following module is missing from the file system:` warnings for eva, conditional fields, entity_rdf, rdfui, rdfx
